Snapchain/cdk-validium-contracts
# Feat: SNA-297 deploy contracts to L1 Testnet Sepolia

## Summary

* Create a new image to deploy contracts to L1 Testnet Sepolia

## Test

1. copy `deployment/deploy_parameters.json.example` to `deployment/deploy_parameters.json` and change the value of 
 `initialCDKValidiumDeployerOwner` with the deployer address;
2. change the `environment` in the `docker-compose.sepolia.yml`:
```
      - MNEMONIC=<deployer-mnemonic>
      - INFURA_PROJECT_ID=<deployer-infura-project-id>
```
3. under the `docker` folder, run `docker-compose up`

**Note**:  The deployer address should be funded with enough Sepolia ETH, and need to change the deployer mnemonic every time testing.
